source/crud/tests/unified/updateOne-sort.yml

```diff
@@ -0,0 +1,125 @@
+description: updateOne-sort
```

Comment:
I assume this file mirrors what already exists in `bulkWrite-updateOne-sort.yml`, so consider my comments there.